test/fixtures/ledger_transaction_fixtures.js

```diff
@@ -0,0 +1,287 @@
+      amount_submitted: opts.refund_summary_submitted || 0
+    },
+    settlement_summary: {
+      capture_submit_time: opts.capture_submit_time || null,
+      captured_date: opts.captured_date || null
+    },
+    card_details: {
+      last_digits_card_number: opts.last_digits_card_number || '0002',
+      cardholder_name: opts.cardholder_name || 'Test User',
+      expiry_date: opts.expiry_date || '08/23',
+      billing_address: {
+        line1: opts.billing_address_line1 || 'address line 1',
+        line2: opts.billing_address_line2 || 'address line 2',
+        postcode: opts.billing_address_postcode || 'AB1A 1AB',
+        city: opts.billing_address_city || 'London',
+        country: opts.billing_address_country || 'GB'
+      },
+      card_brand: opts.card_brand || 'Visa'
+    },
+    delayed_capture: opts.delayed_capture || false
+  }
+
+  if (opts.corporate_card_surcharge) {
+    data.corporate_card_surcharge = opts.corporate_card_surcharge
+    data.total_amount = opts.total_amount
+  }
+  if (opts.fee) data.fee = opts.fee
+  if (opts.net_amount) data.net_amount = opts.net_amount
+  if (opts.wallet_type) data.wallet_type = opts.wallet_type
+  if (opts.metadata) data.metadata = opts.metadata
+
+  return data
+}
+
+const buildTransactionSearchResultWithDefaults = (opts = {}) => {
+  const data = {
+    amount: opts.amount || 20000,
+    state: buildChargeEventStateWithDefaults(opts.state),
+    description: opts.description || 'ref1',
+    reference: opts.reference || 'ref188888',
+    links: [],
+    return_url: 'https://somewhere.gov.uk/rainbow/1',
+    transaction_id: opts.transaction_id || 'ht439nfg2l1e303k0dmifrn4fc',
+    gateway_transaction_id: opts.gateway_transaction_id || '4cddd970-cce9-4bf1-b087-f13db1e199bd',
+    email: opts.email || 'gds-payments-team-smoke@digital.cabinet-office.gov.uk',
+    transaction_type: opts.transaction_type || 'charge',
+    created_date: opts.created_date || '2018-05-01T13:27:00.057Z',
+    card_details: {
+      last_digits_card_number: opts.last_digits_card_number || '0002',
+      cardholder_name: opts.cardholder_name || 'Test User',
+      expiry_date: opts.expiry_date || '08/23',
+      card_brand: opts.card_brand || 'Visa'
+    },
+    delayed_capture: opts.delayed_capture || false
+  }
+
+  if (opts.corporate_card_surcharge) {
+    data.corporate_card_surcharge = opts.corporate_card_surcharge
+    data.total_amount = opts.total_amount
+  }
+  if (opts.fee) data.fee = opts.fee
+  if (opts.net_amount) data.net_amount = opts.net_amount
+  if (opts.wallet_type) data.wallet_type = opts.wallet_type
+  if (opts.metadata) data.metadata = opts.metadata
+
+  return data
+}
+
+module.exports = {
+  validTransactionSummaryResponse: () => {
+    let data = {
+      successful_payments: { count: 1, total_in_pence: 2 },
+      refunded_payments: { count: 3, total_in_pence: 4 },
+      net_income: { total_in_pence: 5 }
+    }
+
+    return {
+      getPactified: () => {
```

Comment:
Looks like a lot of these are anonymous method that could directly 'return' the values
```suggestion
      getPactified: () => pactRegister.pactify(data)
```